Code/Framework/AzToolsFramework/AzToolsFramework/ComponentModes/ShapeTranslationOffsetViewportEdit.h

```diff
@@ -6,26 +6,28 @@
  *
  */
 
+#pragma once
```

Comment:
Good spot! 😬

Reply:
Yeah, I'm kind of surprised this didn't break before :p